make.sh

```diff
@@ -19,5 +19,6 @@ runjob fmt ./ci/sub/bin/fmt.sh &
 if <"$CHANGED_FILES" grep -q '\.\(ts\|tsx\|scss\|css\)$'; then
   runjob lint &
 fi
-runjob build 'yarn package' &
+runjob buildinit 'yarn installdep'
+runjob build 'yarn package'
```

Comment:
if you don't add the `&`, I think it won't run the jobs concurrently

I think it should be like (but i'm not sure)
`runjob (buildinit 'yarn installdep' && build 'yarn package') &`